controllers/rhmi/prometheusRules.go

```diff
@@ -57,7 +57,7 @@ func (r *RHMIReconciler) newAlertsReconciler(installation *integreatlyv1alpha1.R
 						},
 						Expr:   intstr.FromString(fmt.Sprintf(`absent(%s_status{stage='complete'} == 1) and absent(%s_version{version=~".+"})`, installationName, installationName)),
 						For:    "120m",
-						Labels: map[string]string{"severity": "critical", "product": installationName, "addon": "managed-api-service", "namespace": "openshift-monitoring"},
+						Labels: map[string]string{"severity": "critical", "product": installationName, "addon": installation.Spec.Type, "namespace": "openshift-monitoring"},
```

Comment:
installation.Spec.Type would be either `managed` or `managed-api` which doesn't exactly replace the value that is currently present. The two types for addons are RHOAM -> `managed-api-service` and RHMI -> `integreatly-operator`
I'm not sure if these values are consumed by anything or if they are just informational for SRE. The `product` label is using the `installationName` which would contain either rhmi/rhoam.

Reply:
@laurafitzgerald, thank you for pointing the above things.

As a followup fix to that:
- I have added a new internal function `getAddonName()`. This function will return `managed-api-service` for RHOAM and `integreatly-operator` for RHMI.
-  And so, in place of the hardcoded value "addon": "managed-api-service", now the field value will be evaluated using the above function like `"addon": getAddonName(installation)`.

Reply:
@laurafitzgerald good point on `consumed by anything or if they are just informational for SRE`. Do you think it's safe to remove this label? I'm thinking of even removing this label if it's not consumed by anyone since `product` is already available.

Reply:
Hi @laurafitzgerald,

As Jose pointed above, I was also wondering if we could remove `addon` label provided it is not consumed by anything else. Since the `product` label is giving information on what type of addon is installed (`rhoam` or `rhmi`), that might be enough.

Please give your views. Thanks!